src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -826,7 +821,7 @@ private void appendAppUIDToAccount(StorageHelper cryptoHelper, Account account)
             } else {
                 try {
                     appIdList = cryptoHelper.decrypt(appIdList);
-                } catch (Exception ex) {
+                } catch (GeneralSecurityException | IOException ex) {
```

Comment:
@heidijinxujia What do you like to log for `GeneralSecurityException` in there? Will changes in the `Logger.error` would address your concerns here too?